app/models/api/fake_data/activity_report.rb

```diff
@@ -12,7 +12,8 @@ def data
     @data ||= {
       id: identifier,
       attributes: {
-        display_id: display_id
+        display_id: display_id,
+        startDate: datetime_string
```

Comment:
[suggestion (nonblocking)] If you're making any other changes anyway, this should be `startDate: date_string`. Otherwise I can make that change when I merge mine in.